pom.xml

```diff
@@ -28,30 +28,14 @@
         <version.org.skyscreamer>1.5.1</version.org.skyscreamer>
         <version.maven-resources-plugin>3.3.0</version.maven-resources-plugin>
         <version.com.github.eirslett.frontend-maven-plugin>1.12.1</version.com.github.eirslett.frontend-maven-plugin>
-
-        <artifactId.arquillian.jetty>arquillian-jetty-embedded-11</artifactId.arquillian.jetty>
-        <version.arquillian.jetty>1.0.0.Final</version.arquillian.jetty>
-        <version.jetty>11.0.7</version.jetty>
-        <version.resteasy>6.0.0.Final</version.resteasy>
-        <!-- RESTEasy REST Client relocated to another GAV, so these props are for jakarta auto migration -->
-        <groupId.resteasy.client>org.jboss.resteasy.microprofile</groupId.resteasy.client>
-        <artifactId.resteasy.client>microprofile-rest-client</artifactId.resteasy.client>
-        <version.resteasy.client>2.1.0.Final</version.resteasy.client>
         <version.quarkus>2.13.1.Final</version.quarkus>
-        <version.testng>7.4.0</version.testng>
+        <version.testng>7.7.1</version.testng>
+        <version.arquillian.jetty>1.0.0.Final</version.arquillian.jetty>
+        <version.jetty>11.0.14</version.jetty>
 
-        <!--
-            xmlReportPaths must contain an entry for each project that reports coverage.
-            Each module reports execution data in its own target/jacoco.exec. The exec
-            data is then aggregated by testsuite/extra. The list of paths here allows
-            Sonar to find the aggregate report while processing each module.
-          -->
-        <sonar.coverage.jacoco.xmlReportPaths>
```

Comment:
This is specified via the Maven CLI in the Sonar workflow and no longer needed here.

https://github.com/smallrye/smallrye-open-api/blob/bb4f95328f71d49d54e9d24972d0a3575b570e03/.github/workflows/sonar.yml#L117